_docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/system/html.html.twig

```diff
@@ -44,6 +44,67 @@
 <!DOCTYPE html>
 <!--[if IE 8]><html class="ie8 lt-ie9" lang="en"><![endif]--><!--[if IE 9]><html class="ie9" lang="en"><![endif]--><!--[if (gte IE 9)|!(IE)]<!--> <html lang="en"> <!--<![endif]-->
 <head>
+  {% if (rhd_dtm_code) %}
+  <script>
```

Comment:
You're indentation here is off. The closing tag is indented before the end, but this one is not.